app/models/fee.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: fees
+#
+#  id         :bigint           not null, primary key
+#  active     :boolean          default(TRUE)
+#  multiplier :decimal(, )      default(1.0)
+#  seller_id  :bigint
+#
+# Indexes
+#
+#  index_fees_on_seller_id  (seller_id)
+#
+class Fee < ApplicationRecord
+  belongs_to :seller
```

Comment:
should also update Seller to `has_one` Fee?

Reply:
Good catch, will add!

Reply:
I added this but by nature it only "has one" fee since the seller_id is on the fee's table. belongs_to is sufficient enough in this case. has_one is only useful if fee_id existed on the sellers table.